api/tests/routes/backoffice_v3/collective_bookings_test.py

```diff
@@ -283,6 +286,37 @@ def test_list_bookings_more_than_max(self, authenticated_client):
         assert html_parser.count_table_rows(response.data) == 20
         assert "Il y a plus de 20 résultats dans la base de données" in html_parser.extract_alert(response.data)
 
+    def test_additional_data_when_reimbursed(self, authenticated_client, collective_bookings):
+        # give
+        reimbursed = collective_bookings[4]
+        reimbursement_and_pricing_venue = offerers_factories.VenueFactory()
+        pricing = finance_factories.CollectivePricingFactory(
+            collectiveBooking=reimbursed,
+            status=finance_models.PricingStatus.INVOICED,
+            venue=reimbursement_and_pricing_venue,
+        )
+        bank_info = finance_factories.BankInformationFactory(venue=reimbursement_and_pricing_venue)
+        cashflow = finance_factories.CashflowFactory(
+            reimbursementPoint=reimbursement_and_pricing_venue, bankAccount=bank_info, pricings=[pricing]
+        )
+
+        # when
+        with assert_num_queries(self.expected_num_queries):
+            response = authenticated_client.get(
+                url_for(self.endpoint, status=educational_models.CollectiveBookingStatus.REIMBURSED.name)
+            )
+
+        # then
+        assert response.status_code == 200
+        reimbursement_data = html_parser.extract(response.data, tag="tr", class_="collapse accordion-collapse")[0]
+        assert f"Total payé par l'utilisateur : {filters.format_amount(reimbursed.total_amount)}" in reimbursement_data
+        assert f"Date de remboursement : {reimbursed.reimbursementDate.strftime('%d/%m/%Y')}" in reimbursement_data
+        assert f"Nom de la structure : {reimbursed.offerer.name}" in reimbursement_data
+        assert f"Nom du lieu : {reimbursed.venue.name}" in reimbursement_data
+        assert f"Montant remboursé : {filters.format_cents(-reimbursed.pricing.amount)}" in reimbursement_data
```

Comment:
Je ne suis pas trop pour utiliser les `filters.*` dans les tests auto, puisque comme cette fonction n'est pas testée individuellement, si elle a un bug, alors on aura ce bug en _expected_ ET en _actual_, donc on ne détectera pas le problème. Un `assert "Montant remboursé : 12,90 €" in reimbursement_data` (j'ai mis 12,90 au pif) m'irait très bien.